### PR TITLE
Adds the v2.5 EE beta release notes to the changelog

### PR DESCRIPTION
**Pull request by HCloward**
### Reason
v2.5.0.0 beta release changelog updates

### Testing
[2.5.0.0 beta changelog entry](https://deploy-preview-3033--kongdocs.netlify.app/enterprise/changelog/#2500-beta)

### CODE REVIEW

File: app/enterprise/changelog.md [hunk 1]
```diff
@@ -4,6 +4,223 @@ no_search: true
 no_version: true
 ---
 
+## 2.5.0.0 (beta)
```
Comment (mikefero) [inline app/enterprise/changelog.md:7]: nit: In the event we release another beta I think it would be better to suffix with a `1`; WDYT?

```suggestion
## 2.5.0.0 (beta1)
```

**Review by mikefero — APPROVED** [review]
Nice job putting this together!!!

File: app/enterprise/changelog.md (hunk 1 above)
Comment (HCloward) [reply app/enterprise/changelog.md:7]: That works for me - thanks @mikefero

File: app/enterprise/changelog.md [hunk 2]
```diff
@@ -4,6 +4,223 @@ no_search: true
...
+  See the [HMAC Authentication](/hub/kong-inc/hmac-auth) documentation for more information.
+  [#7037](https://github.com/kong/kong/pull/7037)
+- [Syslog](/hub/kong-inc/syslog) (`syslog`) 
+  The Syslog plugin now includes facility configuration options, which are a way for the plugin to group
+  error messages from different sources. See the description for the facility parameter in the
+  Parameters section of the [Syslog documentation](/hub/kong-inc/syslog) for more
+  information. [#6081](https://github.com/kong/kong/pull/6081).
+- [Prometheus](/hub/kong-inc/prometheus) (`prometheus`) The Prometheus plugin now exposes connected data planes'
+  status on the  control plane. New metrics include the following:  `data_plane_last_seen`, `data_plane_config_hash` and `data_plane_version_compatible`. These metrics can be useful for troubleshooting when data planes have inconsistent configurations across the cluster. See the [Available metrics](/hub/kong-inc/prometheus/#available-metrics) section of the Prometheus plugin documentation for more information. [98](https://github.com/Kong/kong-plugin-prometheus/pull/98)
```
Comment (falondarville) [inline app/enterprise/changelog.md:57]:
```suggestion
  status on the  control plane. New metrics include the following:  `data_plane_last_seen`, `data_plane_config_hash` and `data_plane_version_compatible`. These metrics can be useful for troubleshooting when data planes have inconsistent configurations across the cluster. See the [Available metrics](/hub/kong-inc/prometheus/#available-metrics) section of the Prometheus plugin documentation for more information. [#98](https://github.com/Kong/kong-plugin-prometheus/pull/98)
```

File: app/enterprise/changelog.md [hunk 3]
```diff
@@ -4,6 +4,223 @@ no_search: true
...
+- [Zipkin](/hub/kong-inc/zipkin) (`zipkin`) 
+  The Zipkin plugin now includes the following tags: `kong.route`,`kong.service_name` and `kong.route_name`.
+  See the [Spans](/hub/kong-inc/zipkin/#spans) section of the Zipkin plugin documentation for more information.
+  [115](https://github.com/Kong/kong-plugin-zipkin/pull/115)
```
Comment (falondarville) [inline app/enterprise/changelog.md:61]:
```suggestion
  [#115](https://github.com/Kong/kong-plugin-zipkin/pull/115)
```

File: app/enterprise/changelog.md [hunk 4]
```diff
@@ -4,6 +4,223 @@ no_search: true
...
+- [LDAP Authentication](/hub/kong-inc/ldap-auth) (`ldap-auth`) 
+  The LDAP Authentication schema now includes a default value for the `config.ldap_port` parameter
+  that matches the documentation. Before the plugin documentation
+  [Parameters](https://docs.konghq.com/hub/kong-inc/ldap-auth/#parameters)
+  section included a reference to a default value for the LDAP port; however, the default value was not included in the plugin schema.
+  [#7438](https://github.com/kong/kong/pull/7438)
+- [Prometheus](/hub/kong-inc/prometheus) (`prometheus`) 
+  The Prometheus plugin exporter now attaches subsystem labels to memory stats. Before, the HTTP
+  and Stream subsystems were not distinguished, so their metrics were interpreted as duplicate entries by Prometheus.
+  [118](https://github.com/Kong/kong-plugin-prometheus/pull/118)
```
Comment (falondarville) [inline app/enterprise/changelog.md:213]:
```suggestion
  [#118](https://github.com/Kong/kong-plugin-prometheus/pull/118)
```

**Review by falondarville — APPROVED** [review]
Small edits for consistency.

File: app/enterprise/changelog.md (hunk 4 above)
Comment (HCloward) [reply app/enterprise/changelog.md:213]: Good eye @falondarville - thanks!

**Review by tyler-ball — APPROVED** [review]
Enterprise notes LGTM!

File: app/enterprise/changelog.md [hunk 5]
```diff
@@ -4,6 +4,223 @@ no_search: true
 no_version: true
 ---
 
+## 2.5.0.0 (beta1)
+**Release Date** 2021/07/13
+
+### Features
+
+#### Enterprise
+- Updated banner content informing users about free mode to include the benefits of and a link for updating to Konnect.
```
Comment (eskibars) [inline app/enterprise/changelog.md:13]: I'd rephrase this a bit to make it clear it only shows up in free mode (vs "about free mode") and change "updating" to "using" since it's not really an update as folks think about it from a version perspective.

I'd suggest something like "For users running in free mode, updated the top banner to include information on and a link to Konnect."

File: app/enterprise/changelog.md [hunk 6]
```diff
@@ -4,6 +4,223 @@ no_search: true
...
+    - Sort the columns.
+    - Search by ID.
+
+#### Core
+- Control Planes can now send updates to new data planes even if the control planes lose connection to the database.
+  [#6938](https://github.com/kong/kong/pull/6938)
+- Kong now automatically adds `cluster_cert`(`cluster_mtls=shared`) or `cluster_ca_cert`(`cluster_mtls=pki`) into
+  `lua_ssl_trusted_certificate` when operating in Hybrid mode. Before, Hybrid mode users needed to configure
+  `lua_ssl_trusted_certificate` manually as a requirement for Lua to verify the Control Plane’s certificate.
+  See [Starting Data Plane Nodes](/gateway-oss/2.5.x/hybrid-mode/#starting-data-plane-nodes)
```
Comment (eskibars) [inline app/enterprise/changelog.md:30]: For this and subsequent docs links, do we have the documentation in EE as well?  I'd hate to bounce a user from the enterprise changelog over to the OSS docs where they then may have different links

File: app/enterprise/changelog.md [hunk 7]
```diff
@@ -4,6 +4,223 @@ no_search: true
...
+  information. [#6081](https://github.com/kong/kong/pull/6081).
+- [Prometheus](/hub/kong-inc/prometheus) (`prometheus`) The Prometheus plugin now exposes connected data planes'
+  status on the  control plane. New metrics include the following:  `data_plane_last_seen`, `data_plane_config_hash` and `data_plane_version_compatible`. These metrics can be useful for troubleshooting when data planes have inconsistent configurations across the cluster. See the [Available metrics](/hub/kong-inc/prometheus/#available-metrics) section of the Prometheus plugin documentation for more information. [#98](https://github.com/Kong/kong-plugin-prometheus/pull/98)
+- [Zipkin](/hub/kong-inc/zipkin) (`zipkin`) 
+  The Zipkin plugin now includes the following tags: `kong.route`,`kong.service_name` and `kong.route_name`.
+  See the [Spans](/hub/kong-inc/zipkin/#spans) section of the Zipkin plugin documentation for more information.
+  [#115](https://github.com/Kong/kong-plugin-zipkin/pull/115)
+
+#### Hybrid Mode
+- Kong now exposes an upstream health checks endpoint (using the status API) on the data plane for better
```
Comment (eskibars) [inline app/enterprise/changelog.md:64]: Can we link to docs on this?

File: app/enterprise/changelog.md [hunk 8]
```diff
@@ -4,6 +4,223 @@ no_search: true
...
+  observability. [#7429](https://github.com/Kong/kong/pull/7429)
+- Control Planes are now more lenient when checking Data Planes' compatibility in Hybrid mode. See the
+  [Version compatibility](/gateway-oss/2.5.x/hybrid-mode/#version_compatibility)
+  section of the Hybrid Mode guide for more information. [#7488](https://github.com/Kong/kong/pull/7488)
+- This release starts the groundwork for Hybrid Mode 2.0 Protocol. This code isn't active by default in Kong 2.5,
```
Comment (eskibars) [inline app/enterprise/changelog.md:69]: I think we can drop this bullet entirely.  The end user doesn't care because it doesn't affect them.

File: app/enterprise/changelog.md [hunk 9]
```diff
@@ -4,6 +4,223 @@ no_search: true
...
+- Pinned `lua-protobuf` to 0.3.2 (previously unpinned) [#7079](https://github.com/kong/kong/pull/7079)
+
+### Fixes
+
+#### Enterprise
+- Renamed the property identifying control planes in hybrid mode when using Vitals with
+  anonymous reports enabled. Before, users received an error on their control planes about
+  a function that could not be used. 
+- Kong now starts even when `SSL_CIPHER_SUITE` is set to `modern` when the Kong Dev Portal is enabled.
+  Before, Kong would fail to start with an NGINX configuration error concerning the number of arguments
```
Comment (eskibars) [inline app/enterprise/changelog.md:93]: Do we have the text of that error or an identifiable section?  In cases where a user has a problem, they often ctrl+f or google their error code and it's helpful to have some specific text that can be searched

File: app/enterprise/changelog.md [hunk 10]
```diff
@@ -4,6 +4,223 @@ no_search: true
...
+  in 'ssl_ciphers' directives.
+- Updated `lua-resty-openssl` to v0.7.3 which fixes an error users were encountering when trying
```
Comment (eskibars) [inline app/enterprise/changelog.md:95]: Similar question here regarding error message

File: app/enterprise/changelog.md [hunk 11]
```diff
@@ -4,6 +4,223 @@ no_search: true
...
+  [`proxy_stream_access_log`](/gateway-oss/2.5.x/configuration/#proxy_stream_access_log)
+  and [`proxy_stream_error`](/gateway-oss/2.5.x/configuration/#proxy_stream_error) in the Configuration
+  Property Reference for more information. [#7046](https://github.com/kong/kong/pull/7046)
+
+#### Migrations
+- Kong no longer assumes that `/?/init.lua` is in the Lua path when doing migrations. Before, when users created
+  a custom plugin in a non-standard location and set `lua_package_path = /usr/local/custom/?.lua`, migrations failed.
+  Migrations failed because the Kong core file is `init.lua` and it is required as part of `kong.plugins.<name>.migrations`.
+  With this fix, migrations no longer expect `init.lua` to be a part of the path. [#6993](https://github.com/kong/kong/pull/6993)
+- Kong no longer emits errors when doing `ALTER COLUMN` operations in Apache Cassandra 4.0.
```
Comment (eskibars) [inline app/enterprise/changelog.md:178]: Let's add a note here that even with this, Cassandra 4.0 is not yet supported

File: app/enterprise/changelog.md [hunk 12]
```diff
@@ -4,6 +4,223 @@ no_search: true
...
+  The Prometheus plugin exporter now attaches subsystem labels to memory stats. Before, the HTTP
+  and Stream subsystems were not distinguished, so their metrics were interpreted as duplicate entries by Prometheus.
+  [#118](https://github.com/Kong/kong-plugin-prometheus/pull/118)
+- [Zipkin](/hub/kong-inc/zipkin) (`zipkin`)
+  - The plugin now works even when `balancer_latency` is `nil`.
+  - The plugin no longer shares context between several Zipkin plugins. Before the plugin was using `ngx.ctx` exclusively,
+    even in a global context, which meant more than one instance of the Zipkin plugin would override each other. Now the plugin
+    uses `kong.ctx.plugin` to hold the `zipkin` table, instead of `ngx.ctx`.
+- [External plugins](/gateway-oss/2.5.x/external-plugins)
+  This release includes better error messages when external plugins fail to start.
```
Comment (eskibars) [inline app/enterprise/changelog.md:220]: Better how?  Are there any examples?

**Review by eskibars — CHANGES_REQUESTED** [review]
Mostly LGTM.  A few hopefully minor things.  The biggest thing to me is getting these docs links updated to point to /enterprise/ instead of /gateway-oss/ where possible quickly.  Where we have no corresponding enterprise docs, let's fix them before GA

File: app/enterprise/changelog.md (hunk 5 above)
Comment (HCloward) [reply app/enterprise/changelog.md:13]: ok, will update

File: app/enterprise/changelog.md (hunk 6 above)
Comment (HCloward) [reply app/enterprise/changelog.md:30]: At this time there are no Enterprise links bc I do not add Enterprise content until the GA release so as to not confuse users who are not using the beta release. I will be adding this content and it will be available and the links updated for the GA release.

File: app/enterprise/changelog.md (hunk 7 above)
Comment (HCloward) [reply app/enterprise/changelog.md:64]: Yep, will add.

File: app/enterprise/changelog.md (hunk 8 above)
Comment (HCloward) [reply app/enterprise/changelog.md:69]: ok, will delete it

File: app/enterprise/changelog.md (hunk 9 above)
Comment (HCloward) [reply app/enterprise/changelog.md:93]: Yep, will add.

File: app/enterprise/changelog.md (hunk 10 above)
Comment (HCloward) [reply app/enterprise/changelog.md:95]: will do

File: app/enterprise/changelog.md (hunk 11 above)
Comment (HCloward) [reply app/enterprise/changelog.md:178]: will do

File: app/enterprise/changelog.md (hunk 12 above)
Comment (HCloward) [reply app/enterprise/changelog.md:220]: yep, will include

**Review by eskibars — APPROVED** [review]
